redirects.json

```diff
@@ -241,7 +241,7 @@
   },
   {
     "source":"/lib/graphqlapi/create-or-re-use-existing-backend/q/platform/js",
-    "target":"/javascript/build-a-backend/graphqlapi/existing-resources/",
+    "target":"/javascript/prev/build-a-backend/graphqlapi/existing-resources/",
```

Comment:
docs.amplify.aws/lib/graphqlapi/create-or-re-use-existing-backend/q/platform/js is the v5 content, so I changed the target to be a /prev/ link, since the page does not exist in the v6 content.